### DISCUSSION TITLE
Coral Beach

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#67] [Card Balance]
After some back and forth suggestions based on his lore, Roxygen came up with this excelent idea. Im very happy with this suggestion. He will become useful and lore accurate.

Coral Beach
4 6 7 7
UNIQUE CHASSEUR POWERIMMUNITY
1 Action: Discard one of your CHASSEUR and gain NOFATE until end of your turn.

`Coral Beach is man of silence and silencing. Coral usually works with squad of hand-picked Chasseurs and is sent on missions requiring extra discipline and ruthlessness. Coral is well known for his ultimate dedication to fullfilling his missions, using all methods available - even if it means sacrificing numerous friendly troops and innocent bystanders. Coral has developed a special mental resistance to things supernatural after being subject to things and events beyond belief while locked up in a cybertronic testchamber during his childhood. He also has some prototype cybernetics installed to strenghten is mental ability.`

**Comment by david-roxygen** [comment]
OLD CARD
![image](https://user-images.githubusercontent.com/88280663/127775172-e04a3049-7373-4696-8a87-5f530ee14f06.png)

NEW VERSION
![Picture1](https://user-images.githubusercontent.com/88280663/127775185-b69453bb-8bc5-44c6-9124-c8788d364c59.png)

**Comment by david-roxygen** [comment]
Considering game-play only, also a version similar to the current one could be useful (and similar to the old doomtrooper card)

Old card:
Coral Beach (CYBERTRONIC)
F:4 S:6 A:7 V:7
Other CHASSEUR gain +2 to F, S and A.

![coral](https://user-images.githubusercontent.com/88280663/137114463-2252c0e2-f44f-455d-8df7-a3c8657d2cd4.jpg)
